Python/binarysearch.py

```diff
@@ -0,0 +1,28 @@
+#Function to check if the number exists ,the array is sorted
+def search (arr,l,r,x):
+    if l > r:
+        return -1
+    mid = (l + r) // 2
+    if x == arr[mid]:
+        return mid
+    if x < arr[mid]:
+        return search(arr, l, mid-1,x)
+    return search(arr, mid+1, r,x)  
+        
+    
+
+num = input("Enter Number of terms:")#Accepting number of terms in an array
```

Comment:
Comments have a structure of two space after the code and one space after hash please follow that for both files. For example, 
`
print("Hello")  # this is a comment
`